scanpy/_settings.py

```diff
@@ -276,6 +278,24 @@ def figdir(self, figdir: Union[str, Path]):
         _type_check(figdir, "figdir", (str, Path))
         self._figdir = Path(figdir)
 
+    @property
+    def cache_compression(self) -> Optional[str]:
+        """\
+        Compression for `sc.read(..., cache=True)` (default `'lzf'`).
+
+        May be `'lzf'`, `'gzip'`, or `None`.
+        """
+        return self._cache_compression
+
+    @cache_compression.setter
+    def cache_compression(self, cache_compression: Optional[str]):
+        if cache_compression not in {'lzf', 'gzip', None}:
+            raise ValueError(
+                f"Invalid cache_compression={cache_compression} "
+                "∉ {{'lzf', 'gzip', None}}"
```

Comment:
I'd probably go with something like:

```python
"`cache_compression` must be in {{'lzf'. 'gzip', None}}"
```


instead of <math>&#x02209;</math>. It's more ascii and biologist friendly.